### ISSUE TITLE
issue of SSR satistical plotting

### ISSUE THREAD

**Issue by yuandong1111** [XuewenWangUGA/GMATA#8]
All functions are fine, but it fail to output images when I  click the button of  "statistic plotting". I can not find out the reason. Please tell me which step I did wrong. Sincerely grateful.
"
Program starts running...
Plotting statistical graphics. 
 Output files: 
 12 figures ending with .jpg"

**Comment by mehmetosari** [comment]
Did you solve the problem? I am having the same problem. If you did can you help me.

**Comment by yuandong1111** [comment]
> Did you solve the problem? I am having the same problem. If you did can you help me.

yes, I have solved the problem.My English is bad. forgive me. Firstly, make sure you have add the perl, gmata, and R to the system path. Then copy the files from gmata-master folder to Strawberry\perl\bin (address which you add to the system path), copy the "xxxx.ssr,sat2" to Strawberry\perl\bin. Lastly, use cmd  cd"work dir" to switch you work path. then use cmd "perl ssrfig.pl -i xxxxx.ssr.sat2 -f1 10 -f2 20 -f3 20 -f4 10 -f5 20". Wait a few seconds, you can see the picture exist in the folder. There is no need to do this in next time. Just use the gmata graphic interface to finish the functions you want. 

Hope my advice can help you.
![grusmonacha fna Fig E2 TopSsrLength](https://github.com/user-attachments/assets/bc5b55b5-f752-490a-8365-3b8fa93a6b13)

**Comment by mehmetosari** [comment]
Thank you for helping to solve the problem
